src/pages/addCard/components/addCard/CardExpiredDateInput.tsx

```diff
@@ -0,0 +1,24 @@
+import { Input, InputProps } from '@components/ui-kit';
+
+interface CardExpiredDateInputProps {
+	value: string;
+	onChange: InputProps['onChange'];
+}
+
+export default function CardExpiredDateInput({
+	value,
+	onChange,
+}: CardExpiredDateInputProps) {
+	return (
+		<Input.Wrapper>
+			<Input
+				variant="basic"
+				value={value}
+				onChange={onChange}
+				placeholder="MM/YY"
+				width="40%"
+				label={<Input.Label label="만료일" />}
+			/>
+		</Input.Wrapper>
+	);
+}
```

Comment:
```suggestion
interface CardExpiredDateInputProps {
	value: string;
	onChange: InputProps['onChange'];
}

const label = <Input.Label label="만료일" />;

export default function CardExpiredDateInput({
	value,
	onChange,
}: CardExpiredDateInputProps) {
	return (
		<Input.Wrapper>
			<Input
				variant="basic"
				value={value}
				onChange={onChange}
				placeholder="MM/YY"
				width="40%"
				label={label}
			/>
		</Input.Wrapper>
	);
}
```

이렇게 표현할 수도 있답니다!
이럴 경우 어떤 효과가 있을까요?